plugins/modules/win_domain_controller.ps1

```diff
@@ -28,7 +28,7 @@ Function Write-DebugLog {
     }
 }
 
-$required_features = @("AD-Domain-Services","RSAT-ADDS")
+$required_features = @("AD-Domain-Services","RSAT-ADDS", "RSAT-AD-AdminCenter")
```

Comment:
What is the reasoning behind adding `RSAT-AD-AdminCenter`. We didn't do it before for `win_domain` and now it means we have an extra feature that will be added that may not be needed.

Reply:
I am fine with excluding it from both `win_domain` and `win_domain_controller` but a comment in `win_domain` (that I've removed) shows that the original author mistakenly thought it would be included with AD-Domain-Services:
https://github.com/ansible-collections/ansible.windows/blob/32bf7dd8744d4cdc55d1144983a4d35f364a74d8/plugins/modules/win_domain.ps1#L17-L18

Reply:
I've removed RSAT-AD-AdminCenter from both `win_domain` and `win_domain_controller`.